crates/re_space_view_time_series/src/space_view_class.rs

```diff
@@ -14,6 +17,43 @@ use re_viewer_context::{
 
 use crate::visualizer_system::{PlotSeriesKind, TimeSeriesSystem};
 
+// ---
+
+#[derive(Debug, Clone)]
+pub struct TimeSeriesSpaceViewFeedback {
+    /// What was the size of the plot canvas in the last frame?
+    pub plot_canvas_size: egui::Vec2,
+
+    /// What were the plot bounds in the last frame?
+    pub plot_bounds: egui_plot::PlotBounds,
+}
+
+impl Default for TimeSeriesSpaceViewFeedback {
+    fn default() -> Self {
+        Self {
+            plot_canvas_size: Default::default(),
+            plot_bounds: egui_plot::PlotBounds::NOTHING,
+        }
+    }
+}
+
+static TIME_SERIES_SPACE_VIEW_FEEDBACK: Mutex<BTreeMap<SpaceViewId, TimeSeriesSpaceViewFeedback>> =
+    Mutex::new(BTreeMap::new());
+
+impl TimeSeriesSpaceViewFeedback {
+    pub fn insert(space_view_id: SpaceViewId, feedback: TimeSeriesSpaceViewFeedback) {
+        TIME_SERIES_SPACE_VIEW_FEEDBACK
+            .lock()
+            .insert(space_view_id, feedback);
+    }
+
+    pub fn remove(space_view_id: &SpaceViewId) -> Option<TimeSeriesSpaceViewFeedback> {
+        TIME_SERIES_SPACE_VIEW_FEEDBACK.lock().remove(space_view_id)
+    }
+}
```

Comment:
Honestly I think that's fine but if anyone has a nicer idea on how to setup that can of feedback loop back to the visualizer system :man_shrugging: